libs/NfcCoreLib/inc/phLibNfc.h

```diff
@@ -62,6 +62,24 @@ typedef enum
     phLibNfc_SE_ActModeOff     = 0x01,
 }phLibNfc_eSE_ActivationMode;
 
+/**
+* \ingroup grp_nci_nfc
+* \brief NFCEE Power and Link Control Info contains power related identifier..
+*/
+typedef enum phLibNfc_PowerLinkMode
+{
+    /** NFCEE  NFCC decides identifier*/
```

Comment:
Put comments on the same line with values.

Reply:
I missed it. Just updated the branch.